content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -257,6 +257,18 @@ Your applications cannot expect any performance SLAs (disk IOPS for example)
 from local ephemeral storage.
 {{< /caution >}}
 
+
+{{< note >}}
+To make the resource quota work on ephemeral-storage, two things need to be done:
+
+* An admin sets the resource quota for ephemeral-storage in a namespace.
+* A user needs to specify limits for the ephemeral-storage resource in the Pod spec.
+
+If the user doesn't specify the ephemeral-storage resource limit in the Pod spec,
+the resource quota is not enforced on ephemeral-storage.
```

Comment:
Are LimitRanges relevant here? (Can you use a LimitRange to enforce a limit on ephemeral storage, even when people who define Deployments, Jobs etc do not specify any limit).

Reply:
Hi @sftim For more clarification on this  could you please take a look at the comments here.
https://github.com/kubernetes/kubernetes/issues/110956#issuecomment-1216244886

Thanks!!